packages/playground/src/views/settings.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <view-layout>
+    <v-card color="primary" class="d-flex justify-center items-center mt-3 pa-3 text-center">
+      <v-icon size="30" class="pr-3">mdi-cog</v-icon>
+      <v-card-title class="pa-0">Settings</v-card-title>
+    </v-card>
+    <v-card class="my-5"
+      ><v-card-title>Theme</v-card-title> <v-card-text>Pick an application theme!</v-card-text>
+      <form-validator v-model="isValidTheme" valid-on-init>
+        <input-validator :rules="[validateTheme]" #="{ props }" ref="themeInput" :value="selectedTheme">
+          <v-select class="pa-3" :items="themes" v-model="selectedTheme" v-bind="props" />
+        </input-validator>
+      </form-validator>
+      <v-card-actions class="justify-end" :disabled="!isValidTheme">
+        <v-btn :disabled="!isValidTheme" @click="UpdateTheme" class="justify-end ml-auto">Update</v-btn></v-card-actions
```

Comment:
The current flow has 3 different forms which might be a hassle for the user if he wants to make changes in all of them. What if we combine all forms in one?

Reply:
what do you think @AhmedHanafy725 @ehab-hassan ?

Reply:
> The current flow has 3 different forms which might be a hassle for the user if he wants to make changes in all of them. What if we combine all forms in one?

All three sections are separated I don't think the user ll change all altogether also in the future maybe we will have more items  and ll divide them into sections like GitHub or Gitea let's leave it as is